setup.cfg

```diff
@@ -58,6 +58,6 @@ testing =
     isort
 
 docs =
-    Sphinx == 6.2.1
+    Sphinx == 5.1.0
```

Comment:
See https://github.com/nexB/skeleton/pull/74 which was merged before. We probably want to keep this at that version, since that is working well and more recent.

> The minimum version requirement for sphinx was 6.2.1 which was causing build failure in read the docs

I tried this locally and could not reproduce, our other repos are also at this version and doc builds are passing allright. See example here: https://github.com/nexB/license-expression/actions/runs/5080171465/jobs/9126802847?pr=85#step:6:8 
Can you paste the build failure traceback here?

Please also squash your commits if they are for the same reason.

Reply:
Please see the screenshot below for the build failure trace from RTD
![image](https://github.com/nexB/skeleton/assets/12524022/2ec0b18b-c225-4553-82e5-70d8208997a9)

To reproduce the error, just fork the this repo and try to build your own readthedocs.
Locally the build is not failing, but it is failing from RTD.

Reply:
Yup, just saw this in another doc build too, seems like a new issue :thinking:

Reply:
We had RTD builds also configured to show up as checks for pull requests before, they were not working for some oauth issues. They should show up now by default in PRs. If you can squash these last two commits into one, we are good to merge (You'll also see the RTD doc build test here). Thanks